chain/mainnet/mezo_31612-1/genesis.json

```diff
@@ -1 +1,296 @@
+            "account_number": "0",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t",
+            "pub_key": null,
+            "account_number": "1",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk",
+            "pub_key": null,
+            "account_number": "2",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt",
+            "pub_key": null,
+            "account_number": "3",
+            "sequence": "0"
+          },
+          "code_hash": "0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+        }
+      ]
+    },
+    "authz": {
+      "authorization": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "mezo1qcjuy52698gf7xaymfdg00guh3nnse6chm0s2t",
+          "coins": [
+            {
+              "denom": "amezo",
+              "amount": "200000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "mezo188vx087k295fxdlx7f4wkznmghajnf0maysxpe",
+          "coins": [
+            {
+              "denom": "amezo",
+              "amount": "300000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "mezo14ugypmyfn2whx4dhsw4jwflk2x5fx2vkvmjrzk",
+          "coins": [
+            {
+              "denom": "amezo",
+              "amount": "400000000000000000000000000"
+            }
+          ]
+        },
+        {
+          "address": "mezo1lgrfdvcj38gwcwr0qv4ytdfcygc0mg6slsj3yt",
+          "coins": [
+            {
+              "denom": "amezo",
+              "amount": "100000000000000000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [
+        {
+          "denom": "amezo",
+          "amount": "1000000000000000000000000000"
+        }
+      ],
+      "denom_metadata": [],
+      "send_enabled": []
+    },
+    "bridge": {
+      "params": {
+        "max_erc20_tokens_mappings": 20,
+        "btc_supply_assertion_enabled": true
+      },
+      "assets_locked_sequence_tip": "0",
+      "source_btc_token": "0x18084fbA666a33d37592fA2633fD49a74DD93a88",
+      "erc20_tokens_mappings": [],
```

Comment:
Are we going to create them manually later? Who needs to perform this transaction?

Reply:
The PoA owner, see [TET-608: Deploy bridgeable ERC20 on Mezo](https://linear.app/thesis-co/issue/TET-608/deploy-bridgeable-erc20-on-mezo)